src/users/cd-login.vue

```diff
@@ -41,7 +41,11 @@
     },
     methods: {
       async validateForm() {
-        return this.$validator.validateAll();
+        try {
```

Comment:
I get why you put this back in, because one of the tests failed. What should actually be done is the test itself should be removed along with this try/catch as it's no longer valid.